.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
```

Comment:
Perhaps it is just me, but I find this yaml indentation formatting a bit inconsistent. Thoughts on the following, which is what the `pretterier` formatting tool outputs:

```yml
repos:
  - repo: https://github.com/PyCQA/isort
    rev: 5.6.4
    hooks:
      - id: isort
  - repo: https://gitlab.com/pycqa/flake8
    rev: 3.8.4
    hooks:
      - id: flake8
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v3.3.0
    hooks:
      - id: trailing-whitespace
      - id: end-of-file-fixer
      - id: check-case-conflict
      - id: check-json
  - repo: https://github.com/pre-commit/mirrors-eslint
    rev: v7.15.0
    hooks:
      - id: eslint
  - repo: https://github.com/pre-commit/pygrep-hooks
    rev: v1.7.0
    hooks:
      - id: rst-backticks
      - id: rst-directive-colons
      - id: rst-inline-touching-normal
```